docs/spark/how-to-guides/windows-instructions.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Build a .NET for Apache Spark application on Windows
+description: Learn how to build your .NET for Apache Spark application on Windows.
+ms.date: 01/02/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+
+# Learn how to build your .NET for Apache Spark application on Windows
+
```

Comment:
Should I change the title to say that or add that line? Seems like content wise they are saying the same thing?

Reply:
Yeah on second review, maybe you can just leave as-is? The other how-to's include an intro sentence like this, but since it'd end up just duplicating the title essentially, it's probably fine to just ignore this suggestion :)